Content.Server/Explosion/EntitySystems/ExplosionSystem.cs

```diff
@@ -344,19 +344,19 @@ private void FlashInRange(EntityCoordinates epicenter, float flashRange)
 
             // logging
             var text = $"{epicenter} with range {devastationRange}/{heavyImpactRange}/{lightImpactRange}/{flashRange}";
-            if (entity == null)
+            if (entity == null || !entity.Value.IsValid())
             {
                 _logSystem.Add(LogType.Explosion, LogImpact.High, $"Explosion spawned at {text}");
             }
-            else if (user == null)
+            else if (user == null || !user.Value.IsValid())
             {
                 _logSystem.Add(LogType.Explosion, LogImpact.High,
-                    $"{EntityManager.GetEntity(entity.Value)} exploded at {text}");
+                    $"{EntityManager.ToPrettyString(entity.Value)} exploded at {text}");
```

Comment:
i think the string interpolator might automatically toprettystring if you pass it an entityuid, would have to check with vera/smug

Reply:
Sadly you apparently still need to do it explicitly